frontend/src/js/audio-item-template.js

```diff
@@ -84,7 +84,7 @@ export const itemTemplate = (hit, { html, components }) => html`
           >
             <!-- ID -->
             <div title="Unique file identifier">
-              #${hit.objectID.padStart(4, '0')}
+              #${components.Highlight({ hit, attribute: 'archiveId' })}
```

Comment:
Instead of padding with zeros, let's just make this block have a min width, around `5rem` (to accommodate 5 characters), to avoid visual horizontal shift between records.